test/integration/daemonset/daemonset_test.go

```diff
@@ -697,6 +697,15 @@ func TestNotReadyNodeDaemonDoesLaunchPod(t *testing.T) {
 }
 
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	// Look for TestInsufficientCapacityNodeWhenScheduleDaemonSetPodsEnabled, we don't need this test anymore.
```

Comment:
So can we delete it?

Reply:
This is until we graduate the feature to GA, IIUC @k82cn wanted existing code to be tested till we remove it.

Reply:
Please update the comment to reflect this plan.